content/docs/2.0/migration.md

```diff
@@ -87,8 +94,26 @@ spec:
 
 **Example of v2 ScaledJob**
 
-TODO
-
+```yaml
+apiVersion: keda.sh/v1alpha1
```

Comment:
would you mind adding those helper comments on the changed/added/deleted lines?

eg ` #  <--- Property value was changed`

@jeffhollan like them a lot 😄

Reply:
Thanks! Done.